CHANGELOG.md

```diff
@@ -11,7 +11,9 @@ All notable changes to this project will be documented in this file. The format
 ### Backend
 - Fixed:
   -  Poor TypeORM performance in `/applications` endpoint [issue #1131](https://github.com/bloom-housing/bloom/issues/1131)
-  -  Add "ohaFormat" to CSV exporter
+- Added:
+  -  "ohaFormat" to CSV exporter
+  -  HOPWA preference to `ohaFormat` CSV export format
```

Comment:
I think this isn't quite right? Since HOPWA and and OHA are separate? Confused by "Added hopwa to ohaFormat"